src/controllers/movie.js

```diff
@@ -0,0 +1,54 @@
+const express = require('express');
+const router = express.Router();
+const MovieDomain = require('../domains/movie');
+const movieDomain = new MovieDomain();
```

Comment:
Nice job creating an instance of movie domain. I always forget about this and never use it.